Office hours blog draft (#2292)
nf-core/website: src/content/blog/2024/office_hours.mdx

---
title: 'Introducing nf-core office hours'

> subtitle: "A new initative from nf-core! Here's what you can expect."

Reviewer: ```suggestion
subtitle: "A new initiative - weekly drop-in sessions to connect and chat."
```

headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
headerImageAlt: 'Person using macbook pro and holding cappuccino'
announcement:
    text: 'Introducing nf-core office hours'
    start: 2024-01-12T16:14:00+01:00
    end: 2024-01-13T16:14:00+01:00
pubDate: 2024-01-12T16:14:00+01:00
authors:
    - 'christopher-hakkaart'
label:
    - 'community'
---

# nf-core office hours

> Slack serves as an outstanding platform for asynchronous collaboration.
> Nonetheless, there are moments when engaging in live discussions about your code proves beneficial.

Reviewer: Feels a bit wooden currently?
```suggestion
Slack is great, but sometimes you can't beat sitting down and chatting to someone about your code over a cup of coffee.
```

We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.

# What you can expect

> 

Reviewer: ```suggestion

nf-core office hours will be held online at regular times, open to anyone to drop in and out as they'd like.
It can be nice to just sit with some other people working on similar stuff (even virtually).
And of course it's a great opportunity to chat with others about Nextflow and nf-core development.

```

> Office hours will be most useful for “advanced beginners” - people who have grasped the fundamentals of writing nextflow pipelines, and are now trying to make the leap to designing, coding, and contributing nf-core components and pipelines.

Reviewer: ```suggestion
We expect that nf-core office hours will be most useful for “advanced beginners” and up - people who have grasped the fundamentals of writing nextflow pipelines, and are now trying to make the leap to designing, coding, and contributing nf-core components and pipelines.
Please note that it's not designed to be training or mentorships (we have programs for both of those) and there is no guarantee of who will be there and how much help they'll be able to give you.
```

Reviewer 2: Links to the training/mentorships?

However, regardless of what stage you are at, if you think that spending some time talking about developing with nf-core sparks your interest, then the office hours might be just what you're looking for!

> In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code.

Reviewer 2: Did we check with the maintainers team they are OK contributing to this?

Author: I'll circulate in the channel and join the next meeting for feedback.

> They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.

Reviewer 3: do we need to bump the gather license for this?

Author: I think it would be more organic in Gather but I need to check the cost and ask permission to bump it to a paid plan if everyone agrees. I was going to circulate this to the core team (and then maintainers) to ask for feedback.

Reviewer 3: agree that it will be nice in gather, just wanted to double check the number of seats

Reviewer 4: I think it's just 10 with current settings

Author: Yeah, it is 10, I've messaged inquiring about discounts for community projects.

> They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.

Reviewer: ```suggestion
They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.
[Gather](https://gather.town/) is an online collaborate platform, kind of like Pokemon-meets-Zoom.
You can get an invite to the nf-core Gather space by going to the `#gather-town` channel on the nf-core Slack, writing a message and dropping the Gather emoji - you'll get an automated message with details.
For more information, see the [Gather Town bytesize talk](/events/2022/bytesize-37-gathertown).
```

Office hour sessions won't be structured.
Some example of things that might be discussed include:

-   A specific concept doesn't make sense and you'd like to talk it over
-   You're working on a component or pipeline, you're stuck, and you'd like to get unstuck

> -   You'd like to swap code reviews

Reviewer: ```suggestion
-   You'd like to swap code reviews with someone to get your PR merged
```

-   You're excited about code and your colleages are tired of hearing about it

# Schedule

We are starting a 10 week trial of _nf-core office hours_ while we evaluate the program.
We will focus on the European and American time zones initially.

-   TBD
-   TBD
-   TBD

> After the trial we will collect feedback and evaluate if and how we might improve it.

Reviewer: Please join the `#office-hours` channel on the nf-core Slack to get reminders when sessions are about to start,
and discuss the format / provide feedback.

---
title: 'Introducing nf-core office hours'
subtitle: 'A new initiative - weekly drop-in sessions to connect and chat.'
headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
headerImageAlt: 'Person using macbook pro and holding cappuccino'
announcement:
    text: 'Introducing nf-core office hours'

>     start: 2024-01-12T16:14:00+01:00
>     end: 2024-01-13T16:14:00+01:00

Reviewer 5: ```suggestion
    start: 2024-06-14T16:14:00+01:00
    end: 2024-06-15T16:14:00+01:00
```

> pubDate: 2024-01-12T16:14:00+01:00

Reviewer 5: ```suggestion
pubDate: 2024-06-14T16:14:00+01:00
```

They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.
[Gather](https://gather.town/) is an online collaborate platform, kind of like Pokemon-meets-Zoom.
You can get an invite to the nf-core Gather space by going to the `#gather-town` channel on the nf-core Slack, writing a message and dropping the Gather emoji - you'll get an automated message with details.
For more information, see the [Gather Town bytesize talk](/events/2022/bytesize-37-gathertown).

Office hour sessions won't be structured.
Some example of things that might be discussed include:

-   A specific concept doesn't make sense and you'd like to talk it over
-   You're working on a component or pipeline, you're stuck, and you'd like to get unstuck
-   You'd like to swap code reviews with someone to get your PR merged
-   You're excited about code and your colleages are tired of hearing about it

# Schedule

We are starting a 10 week trial of _nf-core office hours_ while we evaluate the program.
We will focus on the European and American time zones initially.

-   TBD
-   TBD
-   TBD

> After the trial we will collect feedback and evaluate if and how we might improve it.

Author: ```suggestion
After the trial we will collect feedback and evaluate if and how we might improve it.

Please join the `#office-hours` channel on the nf-core Slack to get reminders when sessions are about to start,
and discuss the format / provide feedback.
```

---
title: 'Introducing nf-core office hours'

> subtitle: 'A new initiative - weekly drop-in sessions to connect and chat.'

Reviewer 2: ```suggestion
subtitle: 'A new initiative - weekly drop-in sessions to connect and get help.'
```

Or something? It's a bit more than just chatting

Reviewer 5: connect and work together?

Reviewer 5: ```suggestion
subtitle: 'A new initiative - weekly drop-in sessions to connect and work together.'
```

headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
headerImageAlt: 'Person using macbook pro and holding cappuccino'
announcement:
    text: 'Introducing nf-core office hours'

>     start: 2024-06-14T16:14:00+01:00
>     end: 2024-06-15T16:14:00+01:00
> pubDate: 2024-06-14T16:14:00+01:00

Reviewer 5: ```suggestion
    start: 2024-06-13T16:14:00+01:00
    end: 2024-06-15T16:14:00+01:00
pubDate: 2024-06-13T16:14:00+01:00
```

authors:
    - 'christopher-hakkaart'
label:
    - 'community'
---

# nf-core office hours

Slack is great, but sometimes you can't beat sitting down and chatting to someone about your code over a cup of coffee.

> We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.

Reviewer 2: See above, it seems a bit vauge at the moment

Reviewer: I thought it was fine as it was tbh, but the chatgpt one is also fine.

Reviewer 5: ```suggestion
We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers who are looking to get unstuck with their coding, connect with fellow developers, or collaborate on projects.
```

Reviewer 5: ```suggestion
We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect, chat, and work together.
```

In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code.
They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.
[Gather](https://gather.town/) is an online collaborate platform, kind of like Pokemon-meets-Zoom.
You can get an invite to the nf-core Gather space by going to the `#gather-town` channel on the nf-core Slack, writing a message and dropping the Gather emoji - you'll get an automated message with details.
For more information, see the [Gather Town bytesize talk](/events/2022/bytesize-37-gathertown).

Office hour sessions won't be structured.
Some example of things that might be discussed include:

-   A specific concept doesn't make sense and you'd like to talk it over
-   You're working on a component or pipeline, you're stuck, and you'd like to get unstuck
-   You'd like to swap code reviews with someone to get your PR merged
-   You're excited about code and your colleages are tired of hearing about it

# Schedule

> We are starting a 10 week trial of _nf-core office hours_ while we evaluate the program.

Reviewer 5: ```suggestion
We are starting a 8 week trial of _nf-core office hours_ while we evaluate the program.
```

We will focus on the European and American time zones initially.

> -   Mondays, 1 PM Eastern Time

Reviewer 5: ```suggestion
-   Tuesdays, 2 PM Eastern Time
```

They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.
[Gather](https://gather.town/) is an online collaborate platform, kind of like Pokemon-meets-Zoom.
You can get an invite to the nf-core Gather space by going to the `#gather-town` channel on the nf-core Slack, writing a message and dropping the Gather emoji - you'll get an automated message with details.
For more information, see the [Gather Town bytesize talk](/events/2022/bytesize-37-gathertown).

Office hour sessions won't be structured.
Some example of things that might be discussed include:

-   A specific concept doesn't make sense and you'd like to talk it over
-   You're working on a component or pipeline, you're stuck, and you'd like to get unstuck
-   You'd like to swap code reviews with someone to get your PR merged
-   You're excited about code and your colleages are tired of hearing about it

# Schedule

We are starting a 10 week trial of _nf-core office hours_ while we evaluate the program.
We will focus on the European and American time zones initially.

-   TBD
-   TBD

> -   TBD

Reviewer 5: Poll opened, I will give until middle/end of the week before deciding on a day/time.

> -   TBD
> -   TBD
> -   TBD

Reviewer 5: ```suggestion
-   Mondays, 1 PM Eastern Time
-   Thursdays, 11 AM Central European Summer Time
```
